Functions/Quantification/xASL_quant_M0.m

```diff
@@ -387,7 +387,7 @@
 		xlabel('Slice (integer)');
 
 		% Save and close the figure
-		SavePath = fullfile(x.D.M0CheckDir, 'RevertBsupFxControl.jpg');
+		SavePath = fullfile(x.D.M0CheckDir, [x.SUBJECT '_' x.SESSION '_RevertBsupFxControl.jpg']);

```

Comment:
Did you check that x.SUBJECT & x.SESSION are defined at this point in the code? That was just a guess from me, I haven't tested that.

Reply:
Aha ;) Will check and if this works, I will merge But I'm 99% sure it is define, why wouldn't it :)

Reply:
Fixed